### PR TITLE
Stream support

### PR DESCRIPTION
**Pull request by jahangirmohammed**
Support for `InputStream` load.

### CODE REVIEW

File: src/main/scala/com/signalcollect/triplerush/TripleRush.scala [hunk 1]
```diff
@@ -148,10 +151,13 @@ case class TripleRush(
    * The placement hint should ensure that this gets processed on node 0, because the dictionary resides on that node.
    * If you get a serialization error for the dictionary, it is probably due to a problematic placement hint.
    */
-  def load(filePath: String, placementHint: Option[Long] = Some(OperationIds.embedInLong(1))): Unit = {
-    graph.loadGraph(new FileLoader(filePath, dictionary), placementHint)
+  def load(filePath: String, placementHint: Option[Long] = Some(OperationIds.embedInLong(1)), lang: Option[Lang] = None): Unit = {
+    graph.loadGraph(new DataLoader(Left(filePath), dictionary, lang), placementHint)
   }
 
+  def loadStream(inputStream: InputStream, placementHint: Option[Long] = Some(OperationIds.embedInLong(1)), lang: Option[Lang] = None): Unit = {
```
Comment (pstutz) [inline src/main/scala/com/signalcollect/triplerush/TripleRush.scala]: If the format detection does not work on input streams, then we should probably make that parameter non-optional here and delegate to `graph.loadGraph(new DataLoader(..., Some(lang) ...)`

Comment (jahangirmohammed) [inline src/main/scala/com/signalcollect/triplerush/TripleRush.scala]: ok..how about a default TURTLE? or should we just force a language for correctness?

Comment (pstutz) [inline src/main/scala/com/signalcollect/triplerush/TripleRush.scala]: yes, i would probably remove the default and make it mandatory for streams

Comment (jahangirmohammed) [inline src/main/scala/com/signalcollect/triplerush/TripleRush.scala]: probably, just non-optional argument without a default make sense I guess.

**Pull request merged**